src/widgets/input.css

```diff
@@ -72,7 +74,28 @@ input::placeholder {
   border: 0;
   color: #0060df;
   background-color: transparent;
-  padding: 4px 8px;
+  padding: 4px 8px 4px 12px;
+}
+
+.password-text {
+  width: 136px;
```

Comment:
I copied a lot of these properties straight from Zeplin - some of it seems redundant and could probably get scrubbed out.